gulpfile.js

```diff
@@ -2,17 +2,19 @@ const gulp = require("gulp");
 const download = require("gulp-download2");
 const cp = require("child_process");
 
-const groupId = "io.openliberty.tools";
-const version = "1.0-SNAPSHOT";
+const libertyGroupId = "io.openliberty.tools";
+const libertyVersion = "1.0-SNAPSHOT";
+const jakartaGroupId = "org.eclipse.lsp4jakarta";
+const jakartaVersion = "0.0.1-SNAPSHOT";
 var releaseLevel = "snapshots"; //snapshots or releases
 
-const libertyLemminxName = "liberty-langserver-lemminx-" + version + "-jar-with-dependencies.jar";
+const libertyLemminxName = "liberty-langserver-lemminx-" + libertyVersion + "-jar-with-dependencies.jar";
 const libertyLemminxDir = "../liberty-language-server/lemminx-liberty";
-const libertyLSName = "liberty-langserver-" + version + "-jar-with-dependencies.jar";
+const libertyLSName = "liberty-langserver-" + libertyVersion + "-jar-with-dependencies.jar";
 const libertyLSDir = "../liberty-language-server/liberty-ls";
-const lsp4jakartaName = "org.eclipse.lsp4jakarta.jdt.core-0.0.1-SNAPSHOT.jar";
+const lsp4jakartaName = "org.eclipse.lsp4jakarta.jdt.core-" + jakartaVersion + ".jar";
 const lsp4jakartaJdt = "../lsp4jakarta/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core";
-const jakartaLSName = "org.eclipse.lsp4jakarta.ls-0.0.1-SNAPSHOT-jar-with-dependencies.jar";
+const jakartaLSName = "org.eclipse.lsp4jakarta.ls-" + jakartaVersion + "-jar-with-dependencies.jar";
```

Comment:
minor but for consistency, either name all reference  `lsp4jakartaJDT...` and `lsp4jakartaLS...` OR `jakartaJDT...` and `jakartaLS...`, same comment applies to urls below.